_posts/tutorials/2020-08-23-javalin-jte-example.md

```diff
@@ -0,0 +1,508 @@
+---
+layout: tutorial
+title: Rendering jte templates in Javalin
+permalink: /tutorials/jte
+summarytitle: Rendering jte templates in Javalin
+summary: Learn how to do type-safe server side rendering using jte.
+date: 2020-08-23
+author: <a href="https://github.com/casid" target="_blank">Andreas Hager</a>
+language: ["kotlin", "java"]
+github: https://github.com/casid/jte-javalin-tutorial
+---
+
+## Introduction
+Even in the age of reactive web apps, server-side rendering is still the way to go if SEO is a requirement.
```

Comment:
```suggestion
```

Reply:
I would cut this, people who are still doing server side rendering probably know why they want it.

Reply:
You're probably right :-D